Microsoft.AVS.Management/Microsoft.AVS.Management.psd1

```diff
@@ -76,7 +76,7 @@
     # NestedModules = @()
 
     # Functions to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no functions to export.
```

Comment:
In order for this to work, we will need the higher version of the SsoAdmin module so you will need to include that as a dependency in this manifest.